### PR TITLE
print() is a function in Python 3 (again)

### PR DESCRIPTION
**Pull request by cclauss**
Closes #1272.

Just like #1272 but does not touch the scripts/20YY (year) directories?

@tfmorris, @cdrini, @hornc, @mekarpeles

### CODE REVIEW

**Comment by mekarpeles** [comment]
Thank you, I'll have a look today!

File: openlibrary/catalog/amazon/crawl.py [hunk 1]
```diff
@@ -81,7 +82,7 @@ def get(self, url):
 
         line = fp.readline()
         if not line.startswith('HTTP/1.1 200'):
-            print('status:', repr(line))
+            print(('status:', repr(line)))
```
Comment (mekarpeles) [inline openlibrary/catalog/amazon/crawl.py]: I don't think this needs to be a tuple

File: openlibrary/catalog/amazon/amazon_to_arc.py [hunk 2]
```diff
@@ -10,17 +11,17 @@ def get(sock, host, url):
     fp = sock.makefile('rb', 0)
 
     line = fp.readline()
-    print('status:', repr(line))
```
Comment (mekarpeles) [inline openlibrary/catalog/amazon/amazon_to_arc.py]: Isn't this line sufficient as is?

**Comment by cclauss** [comment]
@mekarpeles Reverted the tuple prints.

**Comment by mekarpeles** [comment]
lgtm

**Pull request merged**